### PR TITLE
Add support for setting up the nRFs internal DCDCs

### PR DESCRIPTION
**Pull request by korken89**

### CODE REVIEW

File: embassy-nrf/src/lib.rs [hunk 1]
```diff
@@ -454,6 +473,19 @@ pub fn init(config: config::Config) -> Peripherals {
     r.tasks_lfclkstart.write(|w| unsafe { w.bits(1) });
     while r.events_lfclkstarted.read().bits() == 0 {}
 
+    #[cfg(not(any(feature = "_nrf5340", feature = "_nrf9160")))]
```
[the remark this answers is not in the document]
Comment (korken89) [reply embassy-nrf/src/lib.rs:476]: Will do!

[the remark this answers is not in the document]
Comment (korken89) [reply embassy-nrf/src/lib.rs:502]: I think it should be fixed now, lets wait for CI

[the remark this answers is not in the document]
Comment (korken89) [reply embassy-nrf/src/lib.rs:502]: Seems fixed :)

**Review by Dirbaio — APPROVED** [review]
Thank you!

**Pull request merged**